relayer/config/config.go

```diff
@@ -67,6 +68,9 @@ type Config struct {
 	DeciderURL             string                   `mapstructure:"decider-url" json:"decider-url"`
 	SignatureCacheSize     uint64                   `mapstructure:"signature-cache-size" json:"signature-cache-size"`
 
+	// mapstructure doesn't handle time.Time out of the box so handle it manually
+	EtnaTime time.Time `json:"etna-time"`
```

Comment:
we could take in a string as well and use `time.Parse` in a date foramt

Reply:
I like that the `v.GetTime()` call will handle both RFC3339 format as well as UNIX timestamps cleanly and output it unambiguously when being written out. 

With time.Parse we would need to manually handle conversion when writing it out to disk and reading it separately.